source/distributed/tria.cc

```diff
@@ -1216,6 +1216,30 @@ namespace
               }
           }
 
+        // Special case: if this cell is active we might be a ghost neighbor
+        // to a locally owned cell across a vertex to a finer cell.
+        // Example (M= my, O=dealii_cell owned by somebody else):
```

Comment:
if you're talking about "we" (i.e., "M=my" cell), then the ghost neighbor in the picture is actually a *coarser* cell, not a *finer* cell as stated in the comment.